src/kontrol/solc_to_k.py

```diff
@@ -237,18 +321,20 @@ def __init__(
             contract_digest: str,
             contract_storage_digest: str,
             sort: KSort,
+            devdoc: dict | None,
         ) -> None:
             self.signature = msig
             self.name = abi['name']
             self.id = id
-            self.inputs = tuple(inputs_from_abi(abi['inputs']))
             self.contract_name = contract_name
             self.contract_digest = contract_digest
             self.contract_storage_digest = contract_storage_digest
             self.sort = sort
             # TODO: Check that we're handling all state mutability cases
             self.payable = abi['stateMutability'] == 'payable'
             self.ast = ast
+            self.natspec_values = parse_devdoc('custom:length', devdoc)
```

Comment:
Could we have this be `kontrol:length-equals'? `kontrol` is more informative than `custom`, and we will want to have potentially other ways of limiting lengths (for example, `less-than`).

Reply:
How extensible is this to other annotations?

Reply:
`parse_devdoc` is general enough to extract any key from a method's devdocs. I'll replace `custom:length` with `kontrol:length-equals` for now.

Reply:
@PetarMax seems like the NatSpec struct allows only specific [tags](https://docs.soliditylang.org/en/latest/natspec-format.html#tags). We could use something like `custom:kontrol-length-equals`.